src/utils/helpers.utils.js

```diff
@@ -151,14 +167,17 @@ export function comparePassword(hashedPassword, password) {
  * @returns {string} token
  */
 export function generateToken(payload, expiresIn = '30days') {
-  const token = jwt.sign(payload, 'SECRET_KEY', {
-    expiresIn,
-  });
+  const token = jwt.sign(payload, process.env.SECRET_KEY, { expiresIn });
   return token;
 }
 
+/**
+ *
+ * @param {string} token
+ * @returns {object/null} decoded tokens
+ */
 export const verifyToken = async token => {
-  return await jwt.verify(token, 'SECRET_KEY', (err, data) => {
+  return await jwt.verify(token, process.env.SECRET_KEY, (err, data) => {
```

Comment:
I think this part was working fine before now. Calling `dotenv.config()` inside this file gives you access to all the environment variables inside the .env file

Reply:
Thanks for the explanation.
the `SECRET_KEY` was passed as a string, which makes the secret key not to be dynamic. 
I would be using the destructured variable `SECRET_KEY`.